app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/lao/StateLao.java

```diff
@@ -12,32 +12,32 @@
  */
 public class StateLao extends Data {
 
-    private final String id; //Hash (organizer + creation + name)
+    private final String id;
     private final String name;
     private final long creation;
-    private final long last_modified;
+    private final long lastModified;
```

Comment:
This name should not be changed. It is used by Gson to compute the json property name,
You need to add @SerializeName("last_modified")